pkg/daemon/daemon.go

```diff
@@ -428,7 +466,10 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 		for index, section := range output.sections {
 			if section.sectionName == "[global]" {
 				section.options["message_tag"] = messageTag
-				section.options["uds_address"] = socketPath
+				// if ha profile is mentioned then you don't need uds_address here
+				if _, ok := (*nodeProfile).PtpSettings[PTP_HA_IDENTIFIER]; !ok {
```

Comment:
Can you create a function like (*nodeprofile).getPhc2SysHAConfig() that will parse the settings and return a clean list of HA configs if there are any, or an empty list otherwise?  Would be cleaner than just parsing through the PtpSetttings, and trimming whitespace.  And the more complexity we move out of this function, the better for us

Reply:
again following existing code format; why treat this special ?